.github/workflows/format.yaml

```diff
@@ -0,0 +1,44 @@
+name: Format
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+
+jobs:
+  format:
+    if: github.repository_owner == 'astrolicious'
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: write
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ github.head_ref }}
+
+      - name: Setup PNPM
+        uses: pnpm/action-setup@v3
+
+      - name: Setup Node
+        uses: actions/setup-node@v4
+        with:
+          node-version: 18
+          cache: "pnpm"
+
+      - name: Install dependencies
+        run: pnpm install
+
+      - name: Format code
+        run: pnpm run format
+
+      - name: Commit changes
+        uses: stefanzweifel/git-auto-commit-action@v5
+        with:
+          commit_message: "[ci] format"
+          branch: ${{ github.head_ref }}
+          commit_user_name: houstonlicious
+          commit_user_email: houstonlicious@astro.licius

```

Comment:
How does that work? Is that a real email?

Reply:
honestly I don't know. I copied from a template. I think it's not needed. The name was my idea, it appears as the commit author, as it profile doesn't exists, it just a placeholder. But if it had a profile it could be used to track it commits

Reply:
Oh I think we should do this differently.

Reply:
don't need to, I can just remove and it will default to github-action bot https://github.com/stefanzweifel/git-auto-commit-action

Reply:
https://github.com/vasfvitor/astro-tips.dev/commit/e3630b7a687e61c4443832cb1c56c259a65fd5ff see live here